flight/Modules/Stabilization/stabilization.c

```diff
@@ -423,18 +423,21 @@ static void stabilizationTask(void* parameters)
 					if (reinit)
 						pids[PID_RATE_ROLL + i].iAccumulator = 0;
 
-					if(fabs(stabDesiredAxis[i]) > max_axislock_rate) {
+					if (fabsf(stabDesiredAxis[i]) > max_axislock_rate) {
```

Comment:
Strangely there are several of these throughout our code. I had thought that there would be warnings-- and thus because of ```-Werror``` build failures-- when promoting to a double as an argument to a function, but it turns out that the compiler catches very few of these double promotions.